cmd/bootstrap.go

```diff
@@ -441,7 +444,7 @@ func bootstrapCmdRun(cmd *cobra.Command, args []string) {
 		}
 
 		zap.S().Debugf("Unable to bootstrap node: %s\n", err.Error())
-		zap.S().Fatalf("Failed to bootstrap node. See %s or use --verbose for logs\n", log.GetLogLocation(util.Pf9Log))
+		zap.S().Fatalf("Failed to bootstrap node. %s. See %s or use --verbose for logs\n", err.Error(), log.GetLogLocation(util.Pf9Log))
```

Comment:
```suggestion
		zap.S().Fatalf("Failed to bootstrap node. error: %s. See %s or use --verbose for logs\n", err.Error(), log.GetLogLocation(util.Pf9Log))
```
Lets keep error messages consistent.

Reply:
Will edit `Failed to prepare node, error: %s` this to `Failed to bootstrap node. %s.` to make it consistent as otherwise it's printed as:
```
Failed to bootstrap node. error: Error: Unable to authorise host ...
```

Reply:
Done